helix-view/src/editor.rs

```diff
@@ -497,6 +497,8 @@ pub struct StatusLineConfig {
     pub right: Vec<StatusLineElement>,
     pub separator: String,
     pub mode: ModeConfig,
+    pub diagnostics: Vec<Severity>,
+    pub w_diagnostics: Vec<Severity>,
```

Comment:
```suggestion
    pub workspace_diagnostics: Vec<Severity>,
```

I would prefer a full name here so it's easy to read